src/vitepress/components/VPAlgoliaSearchBox.vue

```diff
@@ -16,7 +16,7 @@ const router = useRouter()
 
 onMounted(() => {
   // this component will only render if user has configured algolia
-  initialize(config.value.algolia!)
+  initialize(toRaw(config.value.algolia!))
```

Comment:
This seems not necessary since `config.value` is already the raw data.

Reply:
Actually, I tried already without but config.value.algolia strangely have its members as proxy.

Reply:
Weird, I also tried without `toRaw` and it works. 🤔

Reply:
If you log the argument passed to initialize function, what do you have? I got Proxy log, and because of that, texts didn't be applied

Reply:
I double-checked again. Yes, you are right, `config.value.algolia` is a proxy, however, passing it into `initialize()` without `toRaw` also works.

Reply:
Changed